lua/telekasten.lua

```diff
@@ -174,13 +174,13 @@ local function daysuffix(day)
 end
 
 local daymap = {
+    "Sunday",
```

Comment:
Have you checked if this breaks `daymap[weekday]` in `CalendarAction()`? It looks like it might.